src/routes/docs.md

```diff
@@ -56,15 +57,16 @@ Example format string: `yyyy-MM-dd HH:mm:ss`
 ## DatePicker
 
 Component with a calendar for choosing a date.
+The component will not assign a date value until a specific date is selected in the picker.
 
 ### Props
 
-| Prop     | Type   | Description                            |
-| :------- | :----- | :------------------------------------- |
-| `value`  | Date   | Date value                             |
-| `min`    | Date   | The earliest year the user can select  |
-| `max`    | Date   | The latest year the user can select    |
-| `locale` | Locale | Locale object for internationalization |
+| Prop                 | Type   | Description                            |
+|:---------------------| :----- | :------------------------------------- |
+| `value`              | Date   | Date value                             |
```

Comment:
```suggestion
| `value`              | Date | null | Date value                             |
```

Reply:
I can't use the pipe character here without it being treated as the table cell delim. in theory you can escape it with \| but my IDE hates me. I just put (or null) for now but if you find a better way to fix please feel free.

Reply:
Ah true. For me `\|` works just fine though

Reply:
as long as github likes it - I'll make the change.

Reply:
it appears that the linter doesnt like it

Reply:
It's fine, you can run `npm run format` and it's fixed